src/dbnode/storage/repair_test.go

```diff
@@ -614,8 +622,14 @@ func TestDatabaseRepairPrioritizationLogic(t *testing.T) {
 					},
 				},
 			},
-			expectedNS1Repair: expectedRepair{xtime.Range{Start: flushTimeStart, End: flushTimeStart.Add(blockSize)}},
-			expectedNS2Repair: expectedRepair{xtime.Range{Start: flushTimeStart, End: flushTimeStart.Add(blockSize)}},
+			expectedNS1Repair: expectedRepair{
+				xtime.Range{Start: flushTimeStart, End: flushTimeStart.Add(blockSize)},
```

Comment:
Nit: as mentioned in chat, if we add keys to all this initializations, we can avoid having to explicitly add new fields to the initialization:

```
expectedRepair{
    // fwiw, I would consider renaming this to just `Range`, since `expectedRepairRange` is a bit redundant. Not actually important, just a general Go style thing ("avoid names that 'stutter'").
    expectedRepairRange: xtime.Range(...)
}
```

Reply:
Done